tests/index.ts

```diff
@@ -0,0 +1,18 @@
+import Ship from "lib/models/ships/model.ship";
+import dotenv from "dotenv";
+import getResolver from "lib/getResolver";
+import { is } from "uvu/assert";
+import path from "path";
+import { suite } from "uvu";
+
+const $: uvu.Test<Record<string, any>> = suite("sample test");
+
+// Setup process environment variables for uvu
+dotenv.config({ path: path.resolve(process.cwd(), ".env.local") });
```

Comment:
I prefer you'd also read directly as well from `process.env` considering we'll be doing a lot of it in CI.

Reply:
Makes sense. I think that means in the Github workflow file, we would need to reference some secret for remote repo. `GITHUB_TOKEN` is already provided so it's not an issue. I added this line so I could add support for environment variables [like Nextjs](https://nextjs.org/docs/basic-features/environment-variables)

Reply:
@sr229 Should be resolved by [f0cce5c](https://github.com/LeNitrous/Nimi/runs/1172775411?check_suite_focus=true#step:5:10), removed `dotenv` dependency and used `REMOTE_REPO` secret provided by @LeNitrous, only issue is the test fail on `pull_request` trigger cause of the current mongodb/typeorm issues and we need to set the secret in Vercel